stat/fetcher.go

```
@@ -86,10 +113,97 @@ func (self *Fetcher) Run() error {
 	go self.RunReserveRatesFetcher()
 	go self.RunTradeLogProcessor()
 	go self.RunCatLogProcessor()
+	go self.RunFeeSetrateFetcher()
 	log.Printf("Fetcher runner is running...")
 	return nil
 }
 
+func (self *Fetcher) RunFeeSetrateFetcher() {
+	client := http.Client{
+		Timeout: 5 * time.Second,
+	}
+	for {
+		self.FetchTxs(client)
+		time.Sleep(self.sleepTime)
+	}
+}
+
+type APIResponse struct {
+	Message string                 `json:"message"`
+	Result  []common.SetRateTxInfo `json:"result"`
+}
+
+func (self *Fetcher) FetchTxs(client http.Client) {
```

This function should return an error, and handle err at the scope calling it.